src/components/pages/ChooseSeatsPage.vue

```diff
@@ -1,50 +1,145 @@
-<script>
-import { defineComponent } from 'vue';
+<script setup>
+import { ref, computed, onBeforeMount } from 'vue';
+import { useRouter } from "vue-router";
+import { useMoviesStore } from '@/stores/moviesStore';
+import { getScreeningById } from '@/api/service/Screenings';
+import SectionTitle from '@/components/common/Section/SectionTitle.vue';
+import TheContainer from '@/components/common/TheContainer.vue';
+import ScreeningsCard from '@/components/views/ScreeningsCard.vue';
 import AppButton from '@/components/common/App/AppButton.vue';
-import { useAuthStore } from '@/stores/authStore';
-
-export default defineComponent({
-  setup() {
-    const auth = useAuthStore();
-    return { auth };
-  },
-  components: { AppButton },
-  computed: {
-    isLoggedIn() {
-      return this.auth.isLoggedIn
-    }
-  },
+import BreadCrumbs from '@/components/common/BreadCrumbs.vue';
+import ProgressStep from '@/components/common/ProgressStep.vue';
+import CinemaHall from '@/components/views/CinemaHall.vue';
+
+const props = defineProps({
+  screeningId: {
+    type: Number,
+    required: true
+  }
+})
+
+const router = useRouter();
+
+const moviesStore = useMoviesStore()
+const movieData = computed(() => moviesStore.getMovieByMovieId(screeningData.value.movie))
+
+const screeningData = ref({});
+const screeningDate = computed(() =>
+  screeningData.value.datetime
+    .substring(0, 10)
+    .split('-')
+    .reverse()
+    .join("/")
+)
+const screeningTime = computed(() => screeningData.value.datetime.substring(11,16))
```

Comment:
you are relying on format of API data very much which seems not to be very stable solution. Maybe it would be better to init `Date` object with API data and then retrieve properly formatted string from there? e.g.
![Screenshot 2022-10-03 at 16 02 57](https://user-images.githubusercontent.com/4105687/193597138-4426c6c0-8802-4bbe-bb22-ca2f63511c8a.png)